androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt

```diff
@@ -54,7 +55,10 @@ class HomeFragment :
 
     private val viewBinding: FragmentHomeBinding by viewBinding(FragmentHomeBinding::bind)
     private val homeViewModel: HomeViewModel by reduxViewModel(this) { viewModelFactory }
-    private val viewStateDelegate: ViewStateDelegate<HomeFeature.HomeState> = ViewStateDelegate()
+    private val homeDelegate: ViewStateDelegate<HomeFeature.HomeState> = ViewStateDelegate()
```

Comment:
Let's keep `viewStateDelegate` or may be `homeViewStateDelegate` beacuse it not  the same delegate as problemsLimitDelegate, it just switch visible views.